### ISSUE TITLE
TypeError on pipelines page

### ISSUE THREAD

**Issue by benjaminapetersen** [openshift/origin-web-console#1247]
Just noticed this while navigating around a new empty project:

![screen shot 2017-02-10 at 1 04 19 pm](https://cloud.githubusercontent.com/assets/280512/22838119/a0d4ef36-ef91-11e6-8a5a-f092dc48ee61.png)

I think its prob a refactor bug, missed rename.
@jwforres @spadgett

**Comment by spadgett** [comment]
Making a P2 for now until, but might be more severe if it breaks links. Either way, I think we should try to fix for 1.5.

**Comment by spadgett** [comment]
This is the only place we call the wrong method:

```
origin-web-console (master) ➜  ag --js fromTemplateURL
app/scripts/controllers/pipelines.js
123:                $scope.createSampleURL = Navigate.fromTemplateURL($scope.projectName, sampleName, sampleNamespace);
```
